test/test_reversible.py

```diff
@@ -0,0 +1,14 @@
+import jax.numpy as jnp
+from diffrax import diffeqsolve, Dopri5, ODETerm, Reversible, SaveAt
+
+
+def f(t, y, args):
+    return -y
+
+
+term = ODETerm(f)
+solver = Reversible(solver=Dopri5(), l=0.999)
+y0 = jnp.array([2.0])
+saveat = SaveAt(ts=jnp.linspace(0, 1, 100))
+sol = diffeqsolve(term, solver, t0=0, t1=1, dt0=0.01, y0=y0, saveat=saveat)
+# print(sol.ys)
```

Comment:
You'll have to put this into the usual pytest form before you make a pull request. That means defining a function whose name starts with `test_` and you'll probably want to put in some assertions. You don't have to call the function inside the file. Just define it an pytest will automatically call all functions starting with `test_`. There's plenty of examples in the other test files. For your own testing, I recommend you instead use Jupyter notebooks which you can keep outside of the repo or on a separate branch.